packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  TriggerStrategy, createTrigger
+} from '@activepieces/pieces-framework';
+import {
+  httpClient,
+  HttpRequest,
+  HttpMethod,
+  AuthenticationType,
+} from "@activepieces/pieces-common";
+import { callClickupGetTask, clickupCommon } from '../common';
+import { ClickupEventType, ClickupWebhookPayload } from '../common/models';
+import { clickupAuth } from "../../";
+
+export const triggerTaskTagUpdated = createTrigger({
+  auth: clickupAuth,
+  name: 'task_tag_updated',
+  displayName: 'Task Tag Updated',
+  description: 'Triggered when a tag is added or removed or renamed on a task.',
+  sampleData: {
+    event: 'taskTagUpdated',
+    history_items: [
+      { 
+        id: '1394258655167106175',
+        type: 1,
+        date: '1378109721053',
+        field: 'status',
+        parent_id: '900900799744',
+        data: {},
+        source: null,
+        user: {
+          id: 55053258,
+          username: 'Activepieces Apps',
+          email: 'apps@activepieces.com',
+          color: '#aa2fff',
+          initials: 'AA',
+          profilePicture: null,
+        },
+        before: null,
+        after: '90040005586783',
+      },
+    ],
```

Comment:
can you please change sampledata for Tag changed webhook? you can use [this](https://clickup.com/api/developer-portal/webhooktaskpayloads/#tasktagupdated-payload) from clickup documnetation.

Reply:
Thanks, fixed, sorry I am a bit rusted.